overrides/scripts/ThermalExpansion.zs

```diff
@@ -370,7 +369,7 @@ recipes.addShaped(<extrautils2:trashcanenergy>, [
 
 //Mana Powder
 recipes.addShaped(<thermalfoundation:material:1028> * 4,[
-	[<thermalfoundation:material:1026>, <thermalfoundation:material:1026>, <thermalfoundation:material:1024>],
+	[<ore:dustAerotheum>, <ore:dustAerotheum>, <thermalfoundation:material:1024>],

```

Comment:
This change seems unrelated to this feature, we should save it for when we do a proper refactoring of the whole file.